src/main/scala/org/dbpedia/spotlight/wikistats/JsonPediaParser.scala

```diff
@@ -168,48 +168,42 @@ class JsonPediaParser(inputWikiDump: String, lang: String)
     Input:  - None
     Output: - RDD with the paragraph links and the text
    */
+
   def getUriParagraphs(): RDD[(String,String)] = {
 
     import org.apache.spark.sql.functions._
 
     dfWikiRDD.select(explode( new Column("paragraphsLink")).as("paraLink"))
-      .select(explode(new Column("paraLink.links.id")).as("id"),new Column("paraLink.paraText"))
+      .select(explode(new Column("paraLink.links.id")).as("id"),new Column("paraLink.paraText").as("para"))
+      .distinct
+      .rdd
       .map(row => (row.getString(0),row.getString(1)))
+      .reduceByKey(SpotlightUtils.stringConcat)
 
   }
 
-  def getRawWikiText(): Unit ={
 
-    dfWikiRDD.select(explode( new Column("paragraphsLink")).as("paraLink"))
-      .select(explode(new Column("paraLink.links")).as("link"),new Column("paraLink.paraText")).collect().foreach(println)
-      //.map(row => (row.getString(0),row.getString(1)))
-  }
-  /*
-   Logic to create the Memory Token Store
-    Input:  - List of all Token types
-    Output: - Memory Store with the Token information
-   */
-  def createTokenTypeStore(tokenTypes: List[TokenType]): MemoryTokenTypeStore =  {
-
-    val tokenTypeStore = new MemoryTokenTypeStore()
-    val tokens = new Array[String](tokenTypes.size + 1)
-    val counts = new Array[Int](tokenTypes.size + 1)
+/*
+  def getRawWikiText(): Unit ={
 
-    println ("Naveen in tokentypes:")
+    val language = lang
+    dfWikiRDD.select(explode( new Column("paragraphsLink")).as("paraLink"),new Column ("wid"),new Column("type"))
```

Comment:
Good to add comments for this chunk of commented code
Or even better just sack it off